File: javaArchive/src/Debt_LRA_Transcript/Debt.java
package Debt_LRA_Transcript;
// Sena VATANSEVER 150119755

import person.Student;

public class Debt {

	double amount;
	Student student;

	public Debt(double amount, Student student) {
		super();
		this.amount = amount;
		this.student = student;
	}

	// Creating get and set methods for variables
	public double getAmount() {
		return amount;
	}

	public Student getStudent() {
		return student;
	}

	public void setAmount(double amount) {
		this.amount = amount;
	}

	public void setStudent(Student student) {
		this.student = student;
	}

}

File: pythonCode/Debt.py
class Debt:

    def __init__(self):
        pass

    # Creating properties for variables
    
    def getAmount(self):
        return self.__amount

    def setAmount(self, amount: float):
        self.__amount = amount

    def getStudent(self):
        try: 
            x = self.__student
        except AttributeError:
            return None
        return self.__student

    def setStudent(self, student):
        self.__student = student

